scripts/falco-probe-loader

```diff
@@ -0,0 +1,408 @@
+				exit 1;
+			fi
+
+			echo "* Extracting kernel sources"
+
+			mkdir kernel-sources && tar xf kernel-sources.tgz -C kernel-sources --strip-components "${STRIP_COMPONENTS}"
+
+			cd kernel-sources
+			export KERNELDIR=`pwd`
+
+			if [[ "${KERNEL_CONFIG_PATH}" == *.gz ]]; then
+			    zcat "${KERNEL_CONFIG_PATH}" > .config
+			else
+			    cat "${KERNEL_CONFIG_PATH}" > .config
+			fi
+
+			echo "* Configuring kernel"
+			customize_kernel_build
+		fi
+
+		echo "* Trying to compile BPF probe ${BPF_PROBE_NAME} (${BPF_PROBE_FILENAME})"
+
+		make -C "/usr/src/${PACKAGE_NAME}-${FALCO_VERSION}/bpf" > /dev/null
+
+		mkdir -p ~/.falco
+		mv "/usr/src/${PACKAGE_NAME}-${FALCO_VERSION}/bpf/probe.o" "${HOME}/.falco/${BPF_PROBE_FILENAME}"
+
+		if [ -n "${BPF_KERNEL_SOURCES_URL}" ]; then
+			rm -r /tmp/kernel
+		fi
+	fi
+
+	if [ ! -f "${HOME}/.falco/${BPF_PROBE_FILENAME}" ]; then
+		local URL
+		URL=$(echo "${SYSDIG_PROBE_URL}/${SYSDIG_REPOSITORY}/sysdig-probe-binaries/${BPF_PROBE_FILENAME}" | sed s/+/%2B/g)
+
+		echo "* Trying to download precompiled BPF probe from ${URL}"
+
+		curl --create-dirs "${FALCO_PROBE_CURL_OPTIONS}" -o "${HOME}/.falco/${BPF_PROBE_FILENAME}" "${URL}"
+	fi
+
+	if [ -f "${HOME}/.falco/${BPF_PROBE_FILENAME}" ]; then
+		if [ ! -f /proc/sys/net/core/bpf_jit_enable ]; then
+			echo "**********************************************************"
+			echo "** BPF doesn't have JIT enabled, performance might be   **"
+			echo "** degraded. Please ensure to run on a kernel with      **"
+			echo "** CONFIG_BPF_JIT enabled and/or use --net=host if      **"
+			echo "** running inside a container.                          **"
+			echo "**********************************************************"
+		fi
+
+		echo "* BPF probe located, it's now possible to start sysdig"
+
+		ln -sf "${HOME}/.falco/${BPF_PROBE_FILENAME}" "${HOME}/.falco/${BPF_PROBE_NAME}.o"
+		exit $?
+	else
+		echo "* Failure to find a BPF probe"
+		exit 1
+	fi
+}
+
+ARCH=$(uname -m)
+KERNEL_RELEASE=$(uname -r)
+SCRIPT_NAME=$(basename "${0}")
+SYSDIG_PROBE_URL=${SYSDIG_PROBE_URL:-https://s3.amazonaws.com/download.draios.com}
+if [ -n "$SYSDIG_PROBE_INSECURE_DOWNLOAD" ]
+then
+	FALCO_PROBE_CURL_OPTIONS=-fsSk
+else
+	FALCO_PROBE_CURL_OPTIONS=-fsS
+fi
+
+MAX_RMMOD_WAIT=60
+if [[ $# -ge 1 ]]; then
+	MAX_RMMOD_WAIT=$1
+fi
+
+if [ -z "${SYSDIG_REPOSITORY}" ]; then
+	SYSDIG_REPOSITORY="stable"
+fi
+
+if [ "${SCRIPT_NAME}" = "falco-probe-loader" ]; then
+        if [ -z "$SYSDIG_VERSION" ]; then
+	    FALCO_VERSION=$(falco --version | cut -d' ' -f3)
+	fi
+	PROBE_NAME="falco-probe"
+	BPF_PROBE_NAME="falco-probe-bpf"
+	PACKAGE_NAME="falco"
+else
+	echo "This script must be called as falco-probe-loader"
+	exit 1
+fi
+
+if [ "$(id -u)" != 0 ]; then
+	echo "Installer must be run as root (or with sudo)."
+	exit 1
+fi
+
+if ! hash curl > /dev/null 2>&1; then
+	echo "This program requires curl"
+	exit 1
+fi
+
+if [ -v SYSDIG_BPF_PROBE ] || [ "${1}" = "bpf" ]; then
```

Comment:
Hmm, this bit might be problematic. All the documentation says to use `SYSDIG_BPF_PROBE` to enable BPF, but we might want to shift to `FALCO_BPF_PROBE` since we now have the advantage of having forked the script off.